Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -354,6 +383,8 @@ def create_pull_request(cls, repository, args, branch_point):
                 sys.stderr.write("Failed to create pull-request for '{}'\n".format(repository.branch))
                 return 1
             print("Created '{}'!".format(pr))
+            if cls.is_revert_commit(commits[0]):
+                cls.add_comment_to_reverted_commit_bug_tracker(repository, args, commits[0])
```

Comment:
We should pass the `pr` object into this function call, `PullRequest.find_existing_pull_request` will make an extra network request otherwise. We could even make it an optional argument and use the network call if our caller didn't specify the object